.pre-commit-config.yaml

```diff
@@ -24,7 +24,7 @@ repos:
     hooks:
       - id: black
   - repo: https://github.com/PyCQA/flake8
-    rev: 5.0.4
+    rev: 6.0.0
```

Comment:
This seems to break our CI. 

```
    Collecting flake8-pyi==22.10.0
      Downloading flake8_pyi-22.10.0-py37-none-any.whl (24 kB)
    
    The conflict is caused by:
        The user requested flake8 6.0.0 (from /home/runner/.cache/pre-commit/repovn0unijf)
        flake8-pyi 22.10.0 depends on flake8<6.0.0 and >=3.2.1
```

Do you know why CI doesn't run in this pull request? Running the CI would have detected this issue.

Reply:
Fixed in #303